pubspec.yaml

```diff
@@ -4,18 +4,19 @@ version: 0.0.2
 homepage: https://github.com/dotintent/flutter-accessorysetup
 
 environment:
-  sdk: '>=3.4.3 <4.0.0'
-  flutter: '>=3.3.0'
+  sdk: '>=3.4.1 <4.0.0'
+  flutter: '>=3.24.5'
 
 dependencies:
+  ffi: ^2.1.3
   flutter:
     sdk: flutter
   plugin_platform_interface: ^2.0.2
 
 dev_dependencies:
```

Comment:
we will prob need also:

-   ffigen: 16.0.0

Reply:
Thanks, I will add them in the next Pr